server/graphql/v2/object/Host.js

```diff
@@ -1076,6 +1076,59 @@ export const GraphQLHost = new GraphQLObjectType({
           return { nodes: vendors, totalCount: count, limit: args.limit, offset: args.offset };
         },
       },
+      potentialVendors: {
```

Comment:
The way it's implemented, this field will miss vendors created through the "Add funds" flow. When using this flow, we create a profile where the host admin is **not** added as an admin of the profile. Instead, we create a user with the provided info (email, name) and add this person as the sole admin of the profile.

We could solve this by relying on the `CreatedByUserId` column of the profile and checking whether the only admin of the profile has confirmed their email.